ci-operator/step-registry/openshift-extended/test/disruptive/openshift-extended-test-disruptive-ref.yaml

```diff
@@ -10,7 +10,7 @@ ref:
     default: |
         MCO|ETCD|PSAP|SDN|STORAGE|API_Server|Authentication|Cluster_Operator
         Cluster_Infrastructure|OLM|Network_Edge|Operator_SDK
-        Workloads|Image_Registry|Container_Engine_Tools
+        Workloads|Image_Registry|Container_Engine_Tools|INSTALLER_GENERAL
```

Comment:
@sgoveas actually no need to add it in disruptive step and longduration step.
as what we design, you will create dedicated job for disruptive (including longduration case implictly) installer case.
when you use this step in prow job, need to override the TEST_SCENARIOS with INSTALLER_DEDICATED, with TEST_FILTERS and TEST_TIMEOUT, like
(it is just example)
```yaml
- as: baremetalds-ipi-installer-fxx-destructive
  cron: 14 22 23 * *
  steps:
    cluster_profile: equinix-ocp-metal-qe
    env:
      DEVSCRIPTS_CONFIG: |
        IP_STACK=v6v4
        MIRROR_IMAGES=true
        INSTALLER_PROXY=true
        NETWORK_TYPE=OVNKubernetes
      TEST_FILTERS: ~ChkUpgrade&;Serial;Disruptive;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
      TEST_SCENARIOS: INSTALLER_DEDICATED
      TEST_TIMEOUT: "90"
    test:
     - chain: cucushift-installer-check-cluster-health
     - ref: idp-htpasswd
     - ref: openshift-extended-test-disruptive
     - ref: openshift-e2e-test-qe-report
    workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn
```
so, do not add INSTALLER_GENERAL into disruptive step and longduraton step as default value.

Reply:
Done